bundles/org.openhab.ui.dashboard/ESH-INF/i18n/dashboard_de.properties

```diff
@@ -6,7 +6,6 @@ index.your-location=Ihr Standort\:
 setup.subtitle=Einrichten
 setup.welcome=Willkommen bei openHAB 2 - Erstmalige Einrichtung
 setup.intro=openHAB stellt viele verschiedene Add-ons bereit. Um einen leichten Einstieg zu erm\u00f6glichen werden folgende vordefinierte Pakete als Ausgangspunkt angeboten.
-setup.check-doc=Bitte besuchen Sie die Online-Dokumentation, um eine <a href\="https\://www.openhab.org/docs/configuration/packages.html" target\="_blank">detaillierte Beschreibung dieser Pakete</a> zu erhalten.
```

Comment:
Why is this translation removed?

Reply:
I guess all of them have not been approved @ Crowdin. And according to the two configurations shown in the screenshot they have been skipped during export.

![bildschirmfoto von 2018-08-28 21-28-14](https://user-images.githubusercontent.com/5131747/44745978-7a513d80-ab09-11e8-89c5-c28f26260f23.png)

Reply:
Looks like approving them fixes it! Perhaps the setting was previously disabled. Do you also speak French or do you know someone who does and can approve the French translations that are removed by this PR? :smiley:

Reply:
Yes, it does. I will observe what happens in the future.

I invited @ghys to be a French proofreader for us. Maybe we can ask @lolodomo for helping us.